stackrox/stackrox
# ROX-20399: Add mutable feature flag for policy criteria modal

## Description

Pre-work for ROX-19032, "Policy Criteria, change from drag-and-drop"

This flag will allow for testing this feature on release builds.

It will also allow us to turn it off and go back to the old drag and drop criteria if needed.


## Checklist

## Testing Performed

### Here I tell how I validated my change

Because this change only adds the feature flag, I plan to deploy the PR build image, and point a UI at it.

Manual tests:

1. With flag off by default and no environmental variable set, observe the response from the `/featureflags` API call and see that this flag is false.
2. With flag off by default but an environmental variable setting it to true, observe the response from the `/featureflags` API call and see that this flag is true.

Response with env off
<img width="1396" alt="Screen Shot 2023-10-24 at 6 17 27 PM" src="https://github.com/stackrox/stackrox/assets/715729/6efedd47-7784-40db-9a91-3dd4d3024087">

Response with env ON
<img width="1396" alt="Screen Shot 2023-10-24 at 6 22 07 PM" src="https://github.com/stackrox/stackrox/assets/715729/571e8bf5-fc31-4c02-a1b8-1ba16eb5cb47">


### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.
